frontend/components/transaction/TransactionFeesTable.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+    <div v-if="isWalletConnected" class="flex flex-col space-y-1 text-gray-700 dark:text-gray-100">
+        <div v-if="swapTransactionFeeETH" class="flex justify-between">
+            <div>Swap Transaction Fee</div>
+            <div>
+                <FormatCurrency :value="swapTransactionFeeETH" :decimals="5" :currency="ETH" />
+            </div>
+        </div>
+        <div v-else-if="bidTransactionFeeETH" class="flex justify-between">
```

Comment:
You currently account for `bid` and `swap` transactions. Do we also want to add support for `surplus` transactions? You can find the types for these [here](https://github.com/sidestream-tech/unified-auctions-ui/blob/main/core/src/types.ts#L191).

You may need to rename `bidTransactionFee` as it would then overlap. We would also need support for the collection fees (See code to determine total fees [here](https://github.com/sidestream-tech/unified-auctions-ui/blob/main/core/src/surplus.ts#L164))

Reply:
Also see lower comment for implementation suggestion.